src/hooks/use-update-demo-site.tsx

```diff
@@ -25,6 +31,7 @@ export const DemoSiteUpdateProvider: React.FC< DemoSiteUpdateProviderProps > = (
 	const { client } = useAuth();
 	const { __ } = useI18n();
 	const [ updatingSites, setUpdatingSites ] = useState< Set< number > >( new Set() );
+	const [ errorSites, setErrorSites ] = useState< Set< number > >( new Set() );
 	const { updateSnapshot } = useSnapshots();
 
```

Comment:
I think it's better to clear the error when the user re-tries and not after one minute.

```suggestion
			if ( errorSites.has( snapshot.atomicSiteId ) ) {
				setErrorSites( ( prev ) => {
					const next = new Set( prev );
					next.delete( snapshot.atomicSiteId );
					return next;
				} );
			}
```

Reply:
Thanks, I made changes in https://github.com/Automattic/studio/pull/977/commits/e57be3afb0d7f6f56e19847f2c6e46788c5d75be